packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -72,13 +72,17 @@ function getNackReconnectInfo(nackContent: INackContent) {
     const canRetry = nackContent.code !== 403;
     const retryAfterMs = nackContent.retryAfter !== undefined ? nackContent.retryAfter * 1000 : undefined;
     return createGenericNetworkError(
-        `nack [${nackContent.code}]`, message, canRetry, retryAfterMs, { statusCode: nackContent.code });
+        `nack [${nackContent.code}]`,
+        message,
+        { canRetry, retryAfterMs },
+        { statusCode: nackContent.code, driverVersion: "" });
```

Comment:
i don't really like empty string here, could driverVersion be defined as `driverVersion: string | undefined` to enforce explicit undefined.